etc/kayobe/pulp.yml

```diff
@@ -23,11 +23,6 @@ stackhpc_release_pulp_url: "https://ark.stackhpc.com"
 stackhpc_release_pulp_username:
 stackhpc_release_pulp_password:
 
-# Client certificates used to access StackHPC Ark repositories.
-# They are trusted by the 'release' cert guard's CA.
-stackhpc_release_pulp_client_cert: "{{ lookup('file', kayobe_config_path ~ '/ansible/certs/ark.stackhpc.com/client-cert.pem') | trim }}"
-stackhpc_release_pulp_client_key: "{{ lookup('file', kayobe_config_path ~ '/ansible/certs/ark.stackhpc.com/client-key.pem') | trim }}"
```

Comment:
Could you remove the placeholder files?

etc/kayobe/ansible/certs/ark.stackhpc.com/client-*.pem

Reply:
Do you want to keep the directory?

Reply:
No - git won't track it if the files are removed.